### PR TITLE
[#10527] Tweak spacing on student view responses 

### LINKED ISSUE #10527: Student view responses: tweak spacing
Use spacing to indicate relationships between elements. Elements should be closer to other related elements than to non-related elements. In this case, `Evaluee/recipient` label should be closer to the names below than the panel above.

image

Also present in V6 but less severe.

### PR DESCRIPTION
**Pull request by zsherry0803**
Fixes #10527

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10121,9 +10121,9 @@
       }
     },
     "lodash": {
-      "version": "4.17.15",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
-      "integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A=="
+      "version": "4.17.20",
```
Comment (ChooJeremy) [inline package-lock.json:10124]: We try to keep each PR addressing 1 issue (and only that issue). While these updates are good (and will probably be done sometime in the future), it isn't needed to solve this issue.

**Review by ChooJeremy — CHANGES_REQUESTED** [review]
I'm afraid your change breaks other pages.

Try browsing to http://localhost:4200/web/instructor/home. The Response Rate shouldn't be this low. (there may be potentially other pages as well using .ngb-tooltip-class)

File: package-lock.json (hunk 1 above)
Comment (zsherry0803) [reply package-lock.json:10124]: Understand that. Instead of changing css of .ngb-tooltip-class, maybe I should just add space above the element in HTML page? This way should not affect other pages.

Comment (ChooJeremy) [reply package-lock.json:10124]: We would prefer not to hardcode styles in this manner, as this styling may be hard to trace down and doesn't scale well. One thing you could try is adding a class to all such "Evaluee/Recipient" text that would be generated and have that class be positioned slightly lower. That way it doesn't affect other items that are not using that class.

Comment (zsherry0803) [reply package-lock.json:10124]: copy that

> We would prefer not to hardcode styles in this manner, as this styling may be hard to trace down and doesn't scale well. One thing you could try is adding a class to all such "Evaluee/Recipient" text that would be generated and have that class be positioned slightly lower. That way it doesn't affect other items that are not using that class.

File: gradle.template.properties [hunk 2]
```diff
@@ -1,13 +0,0 @@
-# Use this property to enable/disable Gradle Daemon (long-lived background process)
```
Comment (ChooJeremy) [inline gradle.template.properties:1]: Why was this file deleted?

File: src/web/app/components/question-submission-form/question-submission-form.component.html [hunk 3]
```diff
@@ -36,10 +36,11 @@
     <tm-constsum-options-question-instruction *ngIf="model.questionType === FeedbackQuestionType.CONSTSUM_OPTIONS" [questionDetails]="model.questionDetails"></tm-constsum-options-question-instruction>
     <tm-constsum-recipients-question-instruction *ngIf="model.questionType === FeedbackQuestionType.CONSTSUM_RECIPIENTS"
                                                  [questionDetails]="model.questionDetails" [numOfRecipients]="model.recipientSubmissionForms.length"></tm-constsum-recipients-question-instruction>
-
-    <p *ngIf="model.recipientType !== FeedbackParticipantType.NONE && model.recipientType !== FeedbackParticipantType.SELF">
-      <span class="ngb-tooltip-class" ngbTooltip="The party being evaluated or given feedback to">Evaluee/Recipient</span>
-    </p>
+    <div class="evaluee-span">
```
Comment (ChooJeremy) [inline src/web/app/components/question-submission-form/question-submission-form.component.html:39]: We're naming a div (not a span) `evaluee-span`? That seems like something very misleading.

File: src/web/app/components/question-submission-form/question-submission-form.component.scss [hunk 4]
```diff
@@ -32,3 +32,8 @@
 .indent {
   margin-left: 1%;
 }
+
+.evaluee-span {
+  position: relative;
+  top: 5px;
```
Comment (ChooJeremy) [inline src/web/app/components/question-submission-form/question-submission-form.component.scss:38]: Can you show a screenshot of a result of this change?

**Review by ChooJeremy — CHANGES_REQUESTED** [review]

**Comment by ChooJeremy** [comment]
Closing due to inactivity.
